RakutenShop/Scenes/DetailedInstructionsView/DetailedInstructionsViewViewModel.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  DetailedInstructionsViewViewModel.swift
+//  RakutenShop
+//
+//  Created by Garnik Ghazaryan on 2/14/20.
+//  Copyright © 2020 Garnik Ghazaryan. All rights reserved.
+//
+
+import ReactiveSwift
+
+protocol DetailedInstructionsViewCoordinatorDelegate: class {
+    
+}
+
+enum InstructionType: String {
+    case ingredient
+    case equipment
+}
+
+class DetailedInstructionsViewViewModel: BaseViewModel {
+    
+    private let context: Context
+    weak var coordinatorDelegate: DetailedInstructionsViewCoordinatorDelegate?
+    
+    let imagesBaseUrl = "https://spoonacular.com/cdn/"
```

Comment:
you told me that should pass this value through coordinators using context

Reply:
This one is different :)